packages/react-components/lib/header-bar.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { createStyles, makeStyles, AppBar, Toolbar } from '@material-ui/core';
+import { TabPanel, TabPanelProps } from './tab-panel';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Badge from '@material-ui/core/Badge';
+
+export interface HeaderBarProps {
+  tabNames: string[];
+  logoPath?: string;
+  tabPanelData?: TabPanelProps[];
+  children?: React.ReactNode;
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+      width: '100%',
+    },
+    logo: {
+      maxWidth: 120,
+      opacity: 1,
+    },
+    tabsContainer: {
+      borderRight: '0.25px solid rgba(251, 252, 255, 0.5)',
+      borderLeft: '0.25px solid rgba(251, 252, 255, 0.5)',
+      flexGrow: 2,
+    },
+    toolbar: {
+      textAlign: 'right',
+      flexGrow: -1,
+    },
+  }),
+);
+
+export const HeaderBar = (props: HeaderBarProps): React.ReactElement => {
+  const { logoPath, tabNames, children, tabPanelData } = props;
+  const classes = useStyles();
+
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.ChangeEvent<unknown>, newValue: number) => {
+    setValue(newValue);
+  };
+
+  const populateTabs = () => {
+    const tabs = tabNames.map((tabName, index) => (
+      <Tab key={index} label={tabName} value={index} aria-label={`tab-${index}`} />
```

Comment:
The aria label should be a human readable representation of the "subject" of the element, so something like "tab-0" doesn't make any sense, you should put the tab name instead. On another note, does the "label" prop not already provide the aria label?

Reply:
I went to check just to be sure, the `label` prop only refers to the output rendered by the tab! there are no aria labels [implemented](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Tab/Tab.js) in the button base of the component.

Reply:
Yeah, it actually just renders the `label` prop "as is".